### ISSUE TITLE
Python 2: complete removal

### ISSUE THREAD

**Issue by clebergnu** [avocado-framework/avocado-vt#2963]
Hi, is there anyone actively using Python 2 with Avocado-VT from the master branch?  I've tried to set that up recently, and failed miserably.

For once, newer aexpect is required (`pass_fds`), but that won't work on Python 3.

With the removal of Python 2, a complete removal of ancient autotest bits can also proceed.

@sathnaga @luckyh PR #2275 is related to this, so your feedback is appreciated.

**Comment by luckyh** [comment]
@clebergnu yes, it's a good chance for the removal of Python2, adding more maintainers for the discussion @kylazhang @chunfuwen @chloerh ..

> With the removal of Python 2, a complete removal of ancient autotest bits can also proceed.

Agree, I will send a patch for this, thanks.

> 
> @sathnaga @luckyh PR #2275 is related to this, so your feedback is appreciated.

ok, let's wait for #2275 being rebased first.

**Comment by chloerh** [comment]
@xiaodwan seems you might be interested too, about our v2v test.

**Comment by chloerh** [comment]
@xiaodwan seems you might be interested too, about our v2v test.

**Comment by xiaodwan** [comment]
v2v won't be affected and it's glad to hear the news.
